packages/babel-jest/yarn.lock

```diff
@@ -1,727 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Does this mean that each package dependencies are not locked by a `yarn.lock` file?